mne/source_estimate.py

```diff
@@ -700,6 +700,13 @@ def __neg__(self):  # noqa: D105
     def __pos__(self):  # noqa: D105
         return self
 
+    def __abs__(self):
+        """Take the absolute value."""
```

Comment:
Okay then let's override `__abs__` in the `VectorSourceEstimate` class to provide a different docstring and then immediately wrap to a `super` call.